Mathlib/Data/Set/Finite/Basic.lean

```diff
@@ -155,14 +155,9 @@ protected alias ⟨_, toFinset_mono⟩ := Finite.toFinset_subset_toFinset
 
 protected alias ⟨_, toFinset_strictMono⟩ := Finite.toFinset_ssubset_toFinset
 
--- Porting note: `simp` can simplify LHS but then it simplifies something
--- in the generated `Fintype {x | p x}` instance and fails to apply `Set.toFinset_setOf`
 @[simp high]
```

Comment:
Do we still need this, then?

Reply:
`simp` does work now without explicitly specifying the extra hyp parameter to `Set.toFinset_setOf` (though I called `simp` directly since `Set.toFinset_setOf` is a simp lemma)